### DISCUSSION TITLE
Getting separate written codes

### DISCUSSION THREAD

**Discussion by Dorane-Star** [Cyfrin/foundry-full-course-cu#3021] [Q&A]
somebody to help how i can get the codes written in full within this platform, if i want to catch up with a section part of the course.
Example: i want to start from lesson 2: memory, storage and calldata intro.

now, how can i start all over from top in terms of getting the codes.

**Comment by EngrPips** [comment]
Hello @Dorane-Star, Here is the link https://github.com/Cyfrin/foundry-full-course-cu to the code section of this GitHub. If you scroll down enough, you will see the course sections as shown below.

![Screenshot 2024-12-10 at 13 45 33](https://github.com/user-attachments/assets/ebf05bb4-a3a3-48da-868c-6feb926e5a5e)

Clicking on any of the sections shown above will take you to the part of the page that contains that section information (as shown in the image below), and there, you will see the link to the GitHub repo that holds the codebase for that section.

![Screenshot 2024-12-10 at 13 47 40](https://github.com/user-attachments/assets/ac593c99-a3b5-46e0-a206-4cb5d677e1d3)

**Comment by Dorane-Star** [comment]
many thanks
